src/CppUTest/TestRegistry.cpp

```diff
@@ -234,20 +234,20 @@ void TestRegistry::shuffleRunOrder(rand_func_t rand_func)
     {
         return;
     }
-    int numTests = getFirstTest()->countTests();
+    const size_t numTests = static_cast<size_t>(getFirstTest()->countTests());
```

Comment:
Isn't it better to fix the return parameter here?

Reply:
From my point of view `countTests()` and `shuffle_list()` should use an unsigened type anyway. But I am not sure if this breaks any code. If you don't see any problems I'll change this. This also get's rid of the cast (https://github.com/cpputest/cpputest/pull/1205#issuecomment-480574854).

Reply:
Yeah, if you could try and see if that leads to problems then that would be great!